_posts/2015-11-09-foreman-community-newsletter-november.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Foreman Community Newsletter - November 2015
+date: '2015-11-09T15:28:00.001+03:00'
+author: Shimon Shtein
+tags:
+- foreman
+- community
+- cockpit
+- salt
+- smart_proxy
+- discovery
+- digitalocean
+- xen
+- ansible
+- chef
+- demo
+- newsletter
+- puppet
+modified_time: '2015-11-09T15:28:00.001+03:00'
+excerpt: |
+  Foreman 1.9.3 was released, Foreman 1.10.0-RC2 is out for testing, new
+  plugins, events.
+---
+
+### Foreman 1.10.0-RC2 available for testing
+
+The latest Release Candidate for Foreman 1.10 has been released. Feedback is very, very useful at this stage, so please do test and report bugs! Here are some of the upcoming highlights:
+
+#### Headline features 
+
+* A new button to rebuild DNS, DHCP and TFTP configs has been added to 
+the UI and API, which will help in cases where records have been lost 
+due to server rebuilds or other issues. 
+* More detailed host statuses are available, with an extensible 
+mechanism for plugins to be able to add to the green/yellow/red status 
+shown on the host list and dashboard. 
+* Lots of host and host group cloning enhancements and fixes are present 
+in this release, so virtual machine details, Puppet class associations 
+and more are copied correctly, among other improvements. 
+* Creating hosts with host groups, but changing and unsetting certain 
+inherited attributes has become easier with new "inherit" buttons. 
+Similarly, passwords already saved can be removed now. 
+* The smart proxy DNS module has been overhauled, and is now extensible. 
+ New PowerDNS and Route 53 plugins are available already! 
+* Parameter management has seen a few improvements, particularly in the 
+Puppet class edit form.  The host and host group parameters tabs have 
+improved tooltips and layout fixes to improve readability. 
+
+More details in the [announcement](https://groups.google.com/forum/#!topic/foreman-users/lMV2QzaHOuI)
+
+### Foreman 1.9.3 released
+
+Latest release in the 1.9 series is [now available](https://groups.google.com/d/msg/foreman-announce/4jKfFum2RrE/8yjimfEICgAJ) with fixes for some new bugs and regressions.
+
+### New and updated plugins this month
+
+* [foreman_cockpit](https://github.com/theforeman/foreman_cockpit) - first release [announcement](https://groups.google.com/forum/#!topic/foreman-users/QEAQjGs1exQ)
+<br>
+![foreman_cockpit](/static/images/blog_images/2015-11-09-foreman-community-newsletter-november/cockpit.gif)
```

Comment:
While I like the *content*, 1.8Mb *might* be a bit large here - it's ok for a one-off, but if we do this a lot, the images dir is going to get large. Maybe a screenshot or two might work as well? @dLobatog I think it was you who asked for more shiny, what do you think?

Also, I wonder if we want to go smaller (gif or screenshot)? Not sure how this might play in an RSS reader...